en/sensor/optical_flow.md

```diff
@@ -36,21 +36,20 @@ The easiest way to calculate the optical flow is to use the [PX4Flow](../sensor/
 
 #### Other Cameras
 
-It is also possible to use a board/quad that has an integrated camera (Bebop2, Snapdragon Flight). 
+It is also possible to use a board/quad that has an integrated camera (Bebop2, Snapdragon Flight).
 For this the [Optical Flow repo](https://github.com/PX4/OpticalFlow) can be used (see also [snap_cam](https://github.com/PX4/snap_cam)).
 
 ### Range Finder
 
 You can use any supported [distance sensor](../sensor/rangefinders.md).
-However we recommend using a LIDAR over a Sonar, because of robustness and accuracy. 
+However we recommend using a LIDAR over a Sonar, because of robustness and accuracy.
 
 ## Estimators
 
 ### Extended Kalman Filter (EKF2)
 
-For Optical Flow fusion using EKF2, the parameter [EKF2_AID_MASK](../advanced_config/parameter_reference.md#EKF2_AID_MASK) must be set to `1`.
+For Optical Flow fusion using EKF2, the second bit of the parameter [EKF2_AID_MASK](../advanced_config/parameter_reference.md#EKF2_AID_MASK) must be enabled (e.g. flow only -> `2`).
```

Comment:
@ChristophTobler Thanks. It is definitely more correct to refer to "bits enabled" rather than a numerical value. Do people tend to index bitmaps from 1 (or zero)? 

Perhaps the last bit might be better as (i.e. to enable just flow the decimal value would be 2).

Reply:
I think it's a bad idea to say bits/numbers in user-facing docs, since QGC exposes either check-boxes or a drop-down list with strings, which are in no way shown to be correlated with these numbers.

Reply:
@mhkabir Good point. I was going to argue that the information about bits is also useful for people working with the parameter via a developer API, but then they will know to follow the link to the parameter. 

How about we add a screenshot. So something like:

> For optical flow fusion using EKF2, set the *use optical flow* flag in the [EKF2_AID_MASK](../advanced_config/parameter_reference.md#EKF2_AID_MASK) parameter, as shown using *QGroundControl* below:
>  ![image](https://user-images.githubusercontent.com/5368500/44197520-2f870b80-a182-11e8-876a-aec579659044.png)

Reply:
Sure, makes sense!

Reply:
Great :-). Are you set up with gitbook enough to do this - or do you want me to look at it?

Reply:
Mh why gitbook? can I not just add the pic and link it to the md file?

Reply:
I only ever use git to add images. If you're able via some other tool - ie the gitbook editor, be my guest :-)